_drafts/_heat/using-ucd-ucdp.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Using IBM UrbanCode Deploy with IBM Blue Box
+featured: no
+weight: 10
+tags: [heat, UCD]
+author: Ying Tang, Pan Xia Zou
+dateAdded: June 7, 2016
+---
+
+IBM UrbanCode Deploy with Patterns (UCDP) is an environment management and deployment solution that you can use to deploy applications to multiple clouds, and manage their lifecycles. 
+Whereas UCDP provides a graphic and text editor for you to design, test, and promote your full-stack environments, IBM UrbanCode Deploy (UCD) is a more complete solution that helps you fully configure your components.
+
+The Heat engine with IBM Blue Box can interact with UCDP and UCD. If you have UCDP and UCD installed on your instance, you can establish a secured connection from your instance to the Heat engine, and then deploy an application quickly on your instance through UCDP and UCD.
+
+You can either install the UCDP and UCD servers on instances in the same Blue Box environment that you want to integrate with, or on external servers outside IBM Blue Box.
+
+The following sections cover both cases by introducing the steps to install UCDP and UCD and then integrate them with IBM Blue Box.
+
+
+## Prerequisites
+
+1. Download the UCDP and UCD installation packages from [IBM Fix Central](http://www-933.ibm.com/support/fixcentral/).
+
+2. To install UCDP and UCD on your instances, you must have two instances that are provisioned on the same cloud environment. You will install UCDP on one instance and UCD on the other. These two instances must have a public floating IP available, and they must share the same IP type.
```

Comment:
can you please elaborate what you mean by 'IP type'? Are you referring to ipv4 vs. ipv6 or they should be on the same subnet?...